### ISSUE TITLE
Feature Request: ":cwindow" to show error list?

### ISSUE THREAD

**Issue by eddieparker** [jaredpar/VsVim#1991]
I love how :cc and :cn/:cp work to move me through my error list.

Is there any chance we could get :cwindow to show the error list, like it would in vim proper?

**Comment by tankorsmash** [comment]
I came here to suggest the same thing. I'm not a C#/F# dev but looking at https://github.com/jaredpar/VsVim/blob/0c8fb6de529708711a97a13a32c88abc111858cb/Src/VsVimShared/VsVimHost.cs#L516-L528

it might be started by

        public override bool QuickFixOpen()
        {
                SafeExecuteCommand(null, "View.ErrorList");
                return true;
        }

then adding the following to https://github.com/jaredpar/VsVim/blob/0c8fb6de529708711a97a13a32c88abc111858cb/Src/VimCore/Interpreter_Interpreter.fs#L1746

    | LineCommand.QuickFixOpen -> x.QuickFixOpen 

and adding the following to  https://github.com/jaredpar/VsVim/blob/0c8fb6de529708711a97a13a32c88abc111858cb/Src/VimCore/Interpreter_Interpreter.fs#L1146

    member x.QuickFixOpen =
        _vimHost.QuickFixOpen

and adding the following to https://github.com/jaredpar/VsVim/blob/447d980da9aa6c761238e39df9d2b64424643de1/Src/VimCore/Interpreter_Parser.fs#L1551

    member x.ParseQuickFixOpen =
        LineCommand.QuickFixOpen()

and adding the following to https://github.com/jaredpar/VsVim/blob/447d980da9aa6c761238e39df9d2b64424643de1/Src/VimCore/Interpreter_Parser.fs#L2080

                | "cwindow" -> x.ParseQuickFixOpen

Hope that isn't too hard to parse, and that someone closer to the project can maybe take it from there. Thanks for considering it!

**Comment by tankorsmash** [comment]
Thanks @jaredpar!
